src/components/AssetsComponent.jsx

```diff
@@ -8,12 +8,16 @@ import SideMenuComponent from '../_components/SideMenuComponent';
 import { getAssetsAction } from '../_actions/assets.action';
 import { createModelNumbers } from '../_actions/modelNumbers.actions';
 import ModelNumberContainer from '../_components/ModelNumber/ModelNumberContainer';
+import CategoryContainer from '../_components/Category/CategoryContainer';
 import '../_css/AssetComponent.css';
 
 export class AssetsComponent extends Component {
-  state = {
-    activePage: 1,
-    limit: 10,
+  constructor(props) {
```

Comment:
No need for the constructor when there's no initial state using props. Please revert to previous code. I.e.
```
state = {
  activePage: 1,
  limit: 10
};

Reply:
done.